migrations/20220607180511-create-comments.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+    up: async(queryInterface, Sequelize) => {
+        await queryInterface.createTable("Comments", {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER,
+            },
+            userId: {
+                type: Sequelize.INTEGER,
+                references: {
+                    model: 'Users',
+                    key: 'id'
+                },
+                onUpdate: 'CASCADE',
+                onDelete: 'SET NULL'
+            },
+            newsId: {
```

Comment:
In the task, a property is assigned as "post_id", and here you put it as "newId". [Task](https://alkemy-labs.atlassian.net/jira/software/c/projects/OT197/boards/277/roadmap?selectedIssue=OT197-79)

Reply:
The post_id property is incorrect, the project doesn't have post model. In a related task the field appears as news_id, so i assumed the field was news_id instead of post_id [https://alkemy-labs.atlassian.net/jira/software/c/projects/OT197/boards/277?modal=detail&selectedIssue=OT197-81](url)